src/lib/pick-layers.js

```diff
@@ -29,6 +29,11 @@ export function pickLayers(gl, {
   }, () => {
     let zOrder = 0;
```

Comment:
I'd move this declaration down closer to the loop (i.e. keep the blend stuff first)